src/large_pages/node_large_page.cc

```diff
@@ -144,43 +132,120 @@ inline uintptr_t hugepage_align_down(uintptr_t addr) {
 #endif  // ifndef ElfW
 #endif  // defined(__FreeBSD__)
 
+#define CONSTRUCT_ONLY(class_name)              \
+  class_name(const class_name&) = delete;       \
+  class_name(class_name&&) = delete;            \
+  void operator= (const class_name&) = delete;  \
+  void operator= (const class_name&&) = delete
```

Comment:
We used to have a `DISALLOW_COPY_AND_ASSIGN` macro for this but we moved to just writing them out by hand, see #26634. Can you do that here too?